run.py

```diff
@@ -28,9 +26,9 @@ def run(pipeline: Pipeline, data: Tuple[Dataset, Dataset]) -> None:
         evaluators=classification_metrics,
         train=True,
         plugins=[
-            WandbPlugin(
-                WandbConfig(project_id="hate-speech-detection"), pipeline.get_configs()
-            )
+            # WandbPlugin(
```

Comment:
I don't want logging, are there better ways to than just commenting this out?

Reply:
You would rather set a boolean in config? We could do that! We could pass it into wandb aswell as a switch.

Reply:
we can leave it like it is, I'm happy either way!